### DISCUSSION TITLE
Feature Request: Ability to select and download multiple images

### DISCUSSION THREAD

**Discussion by ToddCrimson** [element-hq/element-meta#2605] [Ideas]
Dear Developers and fellow enthusiasts,

My use case is simple, and while I see there are discussions on multiple redactions, which seems to be requested since 2017, please treat this separately, as this is a very painful every day use case.

We have a private server, and primarily use Element (ios / android / desktop) apps for all our discussions and calls. 

We share pics and media as well.  
For purposes of this feature request
USE CASE:

- User A sends 31 images to User B
- User B receives the 31 images
- User B can only download ONE image at a time
- User B has to select or scroll each image separately, and keep hitting save on each one, to place them in the device locally.
PS: This is true in rooms or direct chats for pictures or videos.

Requesting a simple ability in a room or private chat, that through the interface, users can select multiple images and save them to their device gallery in one go.

Thank you, 
And once again, requesting that we keep this request separate from other requests. Bunching them together, then this gets deprioritized, and gets bunched with "redaction", which seems to be contested.

Todd

**Comment by cipherswami** [comment]
oh man 1 year later, I don't even see an option to save it to device.

**Reply by cipherswami** [reply]
Sorry my bad, the save button is hidden some where below in the info section

**Comment by tom-ch1** [comment]
Is this a limitation of Element only or a general Matrix issue? This Use-Case seems to be very relevant and the lack of solution ruins all efforts to convince people to use Matrix instead of Whatsapp | Signal | whatever
